### ISSUE TITLE
codelab-elements readme bazel test step failing

### ISSUE THREAD

**Issue by marcacohen** [googlecodelabs/tools#147]
Following the README (and after installed the 0.18.1 version of bazel, per Shawn's advice) but the first command fails for me:

~/go/src/github.com/googlecodelabs/tools/codelab-elements$ bazel test --test_output=all //demo:hello_test
ERROR: Skipping '//demo:hello_test': no such package 'demo': BUILD file not found on package path
ERROR: no such package 'demo': BUILD file not found on package path
INFO: Elapsed time: 0.120s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (0 packages loaded)
FAILED: Build did NOT complete successfully (0 packages loaded)

**Comment by shawnbuso** [comment]
Good catch, thanks! The directory structure for codelab-elements changed when we moved it here, and it looks like not all of the dependencies got updated. PR #148 fixes this.
